src/nbl/asset/interchange/CPLYMeshFileLoader.cpp

```diff
@@ -606,9 +604,9 @@ bool CPLYMeshFileLoader::genVertBuffersForMBuffer(
 	{
 		constexpr std::array<std::pair<uint8_t, std::string_view>, 3> avaiableOptionsForShaders
 		{ 
-			std::make_pair(E_COL, "nbl/builtin/material/debug/vertex_color_debug_shader/specialized_shader"),
-			std::make_pair(E_UV, "nbl/builtin/material/debug/uv_debug_shader/specialized_shader"),
-			std::make_pair(E_NORM, "nbl/builtin/material/debug/normal_debug_shader/specialized_shader")  // TODO: `normal_debug` is a rather bad name
+			std::make_pair(E_COL, "nbl/builtin/material/debug/vertex_color/specialized_shader"),
+			std::make_pair(E_UV, "nbl/builtin/material/debug/vertex_uv/specialized_shader"),
+			std::make_pair(E_NORM, "nbl/builtin/material/debug/vertex_normal/specialized_shader")
```

Comment:
`normal_debug_shader` -> `vertex_normal` needs matching change all around the source code and folders

Reply:
there is only one STL loader using it, but it will get updated soon when I;m done with PLY